src/main/java/mops/infrastructure/database/repositories/DatePollEntryRepositoryManager.java

```diff
@@ -0,0 +1,46 @@
+package mops.infrastructure.database.repositories;
+
+import mops.domain.models.PollLink;
+import mops.domain.models.Timespan;
+import mops.domain.models.datepoll.DatePollEntry;
+import mops.infrastructure.database.daos.TimespanDao;
+import mops.infrastructure.database.daos.datepoll.DatePollDao;
+import mops.infrastructure.database.daos.datepoll.DatePollEntryDao;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+
+@Repository
+public class DatePollEntryRepositoryManager {
+    private DatePollJpaRepository datePollJpaRepository;
+    private DatePollEntryJpaRepository datePollEntryJpaRepository;
+    private UserJpaRepository userJpaRepository;
+    @Autowired
+    public DatePollEntryRepositoryManager(
+            DatePollJpaRepository datePollJpaRepository,
+            DatePollEntryJpaRepository datePollEntryJpaRepository,
+            UserJpaRepository userJpaRepository) {
+        this.datePollJpaRepository = datePollJpaRepository;
+        this.datePollEntryJpaRepository = datePollEntryJpaRepository;
+        this.userJpaRepository = userJpaRepository;
+    }
+
+    /**
+     * Die Methode soll ein DatePollEntryDao Objekt anhand des pollLinks und des Terminvorschlags
+     * heraussuchen. Als Key wird zuerst der pollLink benutzt, um alle zugehoerigen DatePollEntryDaos
+     * zu generieren, danach wird anhand des TimespanObjektes nach dem zugehoerigen Terminvorschlag gesucht.
+     * @param pollLink Das DatePoll Objekt um das es geht.
+     * @param datePollEntry Der zugehörige Terminvorschlag.
+     * @return DatePollEntryDao Das DatePollEntry Objekt aus der Datenbank.
+     */
+     public DatePollEntryDao findDatePollEntryDao(
```

Comment:
wenn ihr wollt das das hier von der Business Logik aufgerufen wird 2 Sachen: 1. gebt kein Dao zurück sonder wenn schon bitte das passende Geschäftsmodell (In dem Fall DatePollBallot) 2. wir haben ja schon darüber geredet, dass dies für die Konsistenz Grenzen unserer Aggregate unvorteilhaft ist. Das ist jetzt nicht schlimm weil wir diese ausnähme handeln können aber bitte dokumentiert das in euren Entscheidungen ehrlich: also das es keinen Vorteil bringt sondern einfach bei der Entwicklung versehentlich passiert ist.